src/web_app/js/infobox.js

```diff
@@ -115,8 +115,19 @@ InfoBox.prototype.updateInfoDiv = function() {
       remoteAddrType = activeCandPair.stat('googRemoteCandidateType');
     }
     if (localAddr && remoteAddr) {
+      var localCandId = activeCandPair.stat('localCandidateId');
+      var localCand;
+      var localTypePref;
+      if (localCandId) {
+        localCand = getStatsReport(this.stats_, 'localcandidate', 'id',
+            localCandId);
+        if (localCand) {
+          localTypePref = localCand.stat('priority') >> 24;
+        }
+      }
       contents += this.buildLine_('LocalAddr', localAddr +
-          ' (' + localAddrType + ')');
+          ' (' + localAddrType + (localTypePref !== undefined ?
```

Comment:
i don't think this makes a difference. Actually this might just do a non-false check, i.e. 
+ (localTypePref ? formatTypePref(localTypePref) : '')